ApnsPHP/Message/PushType.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * APNS Push Type
+ *
+ * SPDX-FileCopyrightText: Copyright 2024 Move Agency Group B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+namespace ApnsPHP\Message;
+
+/**
+ * APNS Push Type.
+ */
+enum PushType: string
+{
+    /**
+     * The push type for notifications that trigger a user interaction — for example, an alert, badge, or sound.
+     * If you set this push type, the apns-topic header field must use your app’s bundle ID as the topic.
+     */
+    case Alert = 'alert';
+
+    /**
+     * The push type for notifications that deliver content in the background, and don’t trigger any user interactions.
+     * If you set this push type, the apns-topic header field must use your app’s bundle ID as the topic.
+     */
+    case Background = 'background';
+
+    /**
+     * The push type for notifications that request a user’s location. If you set this push type, the apns-topic header
+     * field must use your app’s bundle ID with `.location-query` appended to the end.
```

Comment:
> bundle ID with `.location-query` appended to the end.

Do we want to make ApnsPHP do this?

Reply:
Not sure. At least not as part of this pull request. The issue is that the apns-topic header isn't always constructed from the bundle ID. Doesn't make it impossible, but warrants more careful thinking about the implementation of such.

Reply:
We could validate the header at least. Doesn't have to be in this PR, that's why I did approve + comment